catalog.json

```diff
@@ -54,6 +60,12 @@
     "type": "transformer",
     "demo": true
   },
+  {
+    "image": "gcr.io/kpt-functions/helm-template",
+    "source": "https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/helm-template/run.sh",
+    "description": "Render chart templates locally using helm template. (See [usage](https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/helm-template/usage.md)).",
```

Comment:
You want to have relative link, so that with GithHub pages, the links are rendered to HTML page instead of MD

Reply:
Done